CMakeLists.txt

```diff
@@ -100,7 +100,8 @@ if (${TEST})
     set(TEST_FILES
         test/common/util.c
         test/fixtures/cpu_mmu.c
-        test/alu_8bit_sub_test.c)
+        test/alu_8bit_sub_test.c
+        test/alu_8bit_lds_test.c)
```

Comment:
Is there a reason why the test has `lds` in the name?

Reply:
Yes, as these simple load instructions can easily be tested via a parameterized test we can put all of them into one file without bloating it that much.
I omitted testing all of the load instructions here to keep the chunk of code to review manageable!